agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/PreAggregatedMetricsTest.java

```diff
@@ -132,9 +130,7 @@ void generateHttpClientMetrics() {
                                             exemplar ->
                                                 exemplar
                                                     .hasTraceId("ff01020304050600ff0a0b0c0d0e0f00")
-                                                    .hasSpanId("090a0b0c0d0e0f00")))),
-            metric -> assertThat(metric).hasName("http.client.request.size"),
-            metric -> assertThat(metric).hasName("http.client.response.size"));
```

Comment:
upstream behavior change (but we don't want to be reporting these metrics anyways)